src/store/modules/product/ProductState.ts

```diff
@@ -1,6 +1,8 @@
 export default interface ProductState {
+  cached: any;  
   products: {
     list: any;
     total: number;
   }
+  productsInformation: any;
```

Comment:
I think this state is not needed.

Reply:
Sir according to me this statement is removed in earlier commits please check those